lib/config/decrypt.js

```diff
@@ -1,4 +1,4 @@
-const is = require('@sindresorhus/is');
+const is = require('@sindresorhus/is').default;
```

Comment:
This is because the that module is a modern module. It will work the old way, because they add a compabillity export.

see:
https://github.com/sindresorhus/is/blob/master/source/index.ts#L388-L390